ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/xpn/provision/cucushift-installer-rehearse-gcp-upi-xpn-provision-chain.yaml

```diff
@@ -6,6 +6,7 @@ chain:
   - chain: ipi-conf-gcp
   - ref: upi-conf-gcp
   - ref: upi-install-gcp
+  - ref: gcp-provision-iam-creds-xpn-roles
```

Comment:
We also need to update `cucushift-installer-rehearse-gcp-upi-deprovision` accordingly, right?

Reply:
No. Only for GCP shared VPC installations. BTW, our IPI XPN tests already have the steps.

Reply:
Oh, I meant `cucushift-installer-rehearse-gcp-upi-xpn-provision` is referenced in `cucushift-installer-rehearse-gcp-upi-xpn` workflow, here `gcp-provision-iam-creds-xpn-roles` is added into the provision chain, accordingly the deprovision chain for `cucushift-installer-rehearse-gcp-upi-xpn` should be updated accordingly to reference `gcp-deprovision-iam-creds-xpn-roles`, right?

Reply:
Got it, updated the workflow `cucushift-installer-rehearse-gcp-upi-xpn`, thanks!